lib/fourOhFour.js

```diff
@@ -147,7 +147,7 @@ function fourOhFour (options) {
     this[kFourOhFourLevelInstance][kFourOhFourContext] = context
 
     router.all(prefix + (prefix.endsWith('/') ? '*' : '/*'), routeHandler, context)
-    router.all(prefix || '/', routeHandler, context)
```

Comment:
The slash is assigned to `prefix` as a default a few lines before already, so this branch is never taken.